apps/dashboard/pages/api/v1/github/verify.ts

```diff
@@ -0,0 +1,217 @@
+import { Buffer } from "node:buffer";
+import crypto from "node:crypto";
+import { db } from "@/lib/db";
+import { env } from "@/lib/env";
+
+import type { Readable } from "node:stream";
+import { clerkClient } from "@clerk/nextjs";
+import { sha256 } from "@unkey/hash";
+import { Resend } from "@unkey/resend";
+import type { NextApiRequest, NextApiResponse } from "next";
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+  runtime: "nodejs",
+};
+const { RESEND_API_KEY, GITHUB_KEYS_URI } = env();
```

Comment:
This needs to happen in the handler, otherwise the build will fail if either env var is not defined.

Reply:
Move to handler